src/Footer/Footer.js

```diff
@@ -39,7 +63,11 @@ const Footer = ({ resetDetails, form }) => {
             id="allow-access"
             type="submit"
             buttonStyle="primary mega"
-            onClick={() => {}}
+            onClick={(e) => {
+              e.preventDefault();
+              handleAccessLog(ALLOWED);
+            }}
```

Comment:
Could you create a separate function out of `return` statement as we discussed before?